src/Movies.json

```diff
@@ -0,0 +1,58 @@
+[
```

Comment:
To my mind this file should be in the ** _ consts folder _ **, but not in the root src, are you agree with me?
and if it's const file than no need to name it with capital letter